ngraph/core/src/op/broadcast.cpp

```diff
@@ -269,8 +269,31 @@ op::v1::Broadcast::Broadcast(const Output<Node>& arg,
 
 void op::v1::Broadcast::validate_and_infer_types()
 {
-    util::BroadcastBase::validate_and_infer_types();
+    // m_type is deduced and not always explicitly stated, for cases where broadcast
+    // has 2 inputs its always NUMPY mode
+    if (m_broadcast_spec.m_type == AutoBroadcastType::NONE && get_input_size() < 3)
+    {
+        m_broadcast_spec.m_type = AutoBroadcastType::NUMPY;
+    }
+
+    // Mocking axes_mapping input for cases that don't require it
+    if (m_broadcast_spec.m_type == AutoBroadcastType::NUMPY)
+    {
+        if (get_input_size() < 3)
+        {
+            auto output = op::v0::Constant::create(element::u8, Shape{}, {0})->output(0);
```

Comment:
Why do we need this code?

Reply:
Broadcast op has a default constructor, and two other ones (one that has 2 and one has 3 inputs). 
If we're creating this op through the 2-input constructor, we're also doing this mocking inside the constructor.
But, in this case we're dealing with creating broadcast op through the default constructor, and we have two inputs (so the third input should be mocked but is not), so we're mocking it, you can take look at the v1::broadcast op's constructors to see what I mean.